src/static/css/2019.css

```
@@ -30,6 +30,14 @@ a:focus, a:hover {
   text-decoration: underline;
 }
 
+/* The main wrapper, center aligned */
+.container {
+  max-width: 1280px;
```

Why 1280 when previously it was 1324? Means all the chapters are a little bit thinner and longer.

```suggestion
  max-width: 1324px;
```

---

As I wrote [here](https://github.com/HTTPArchive/almanac.httparchive.org/issues/480#issuecomment-556009375), the Methodology page was relying on a `max-width: 1324px` while Table of Contents & Contributors pages on `max-width: 1024px`. 

While I thought 1324px is an arbitrary value chosen for a breakpoint and 1024px way too small, considering a 1200px is recommended value for desktop-and-up displays when it comes to how [frameworks structure their breakpoints](https://getbootstrap.com/docs/4.3/layout/overview/#responsive-breakpoints) I decided to go with a median to say so between the 1200 which I well guessed you will consider being too low and the actual 1324.

I'm happy to update this to 1324 if that's what you want but I still consider this value is "hardcoded" and too big.

---

OK I hadn't considered the chapters changing. I'm OK with 1200.